src/main/webapp/js/menu.js

```diff
@@ -0,0 +1,70 @@
+const menuTrigger = document.querySelector('.menu-trigger')
+const menu = document.querySelector('.menu')
+const menuPanel = '.menu-panel'
+const menuItem = '.menu-pages-item'
+
+document.onkeydown = function(evt) {
+    evt = evt || window.event;
+    let isEscape = false;
```

Comment:
why not make this into  `isEscape = (evt.keyCode === 27);` so you don't need the extra else?

Reply:
good point, I think my brain was turned off then